app/api/db/db.py

```diff
@@ -0,0 +1,88 @@
+
+from datetime import datetime, timezone
+import os
+import uuid
+from aiohttp import ClientError
+from boto3.dynamodb.conditions import Key
+import boto3
+
+class DatabaseOperations:
+    def __init__(self):
+            self.dynamodb = boto3.resource('dynamodb', region_name=os.environ.get("AWS_REGION"))
+            self.football_results = self.dynamodb.Table('bs-football-results')
+            self.football_context_prompts = self.dynamodb.Table('bs-football-context-prompts')
+            self.user_contacts = self.dynamodb.Table('bs-user-contacts')
+
+    @classmethod
+    def save_prediction(cls, address: str, prediction: str, team: str):
+        """
+        Save prediction to DynamoDB
```

Comment:
No need to address in this PR, but I would use this more _pythonic_ style for documenting:
```
"""
<Brief description>

Parameters:
    @param (param type): <Param description>
    (...)

    Returns:
    <Return description>
"""
```
You can checkout [this code](https://github.com/brainstems/inference-server/blob/main/src/model_operations.py#L9) for reference.